packages/libraries/pg-event-sourcing/src/load/loadFromStream.integration.ts

```diff
@@ -11,59 +14,95 @@ describe(`loadFromStream`, () => {
 
   it(`Étant donné des événements dans un stream,
       Lorsqu'on charge un stream
-      Alors les événements devrait être récupérer dans l'ordre createdAt + version`, async () => {
-    const streamId = 'string#string';
-    const createdAt = new Date().toISOString();
-
-    await executeQuery(
-      `
-      insert
-      into event_store.event_stream
-      values (
-          $1, 
-          $2, 
-          $3, 
-          $4,
-          $5
-          )`,
-      streamId,
-      createdAt,
-      'event-2',
-      2,
-      { test2: '2' },
-    );
-
-    await executeQuery(
-      `
-      insert 
-      into event_store.event_stream 
-      values (
-          $1, 
-          $2, 
-          $3, 
-          $4,
-          $5
-          )`,
-      streamId,
-      createdAt,
-      'event-1',
-      1,
-      { test1: '1' },
-    );
-
-    const actuals = await loadFromStream(streamId);
-
-    expect(actuals).toEqual([
-      {
-        type: 'event-1',
-        version: 1,
-        payload: { test1: '1' },
-      },
-      {
-        type: 'event-2',
-        version: 2,
-        payload: { test2: '2' },
-      },
-    ]);
+      Alors les événements devrait être récupérer dans l'ordre de création et de versionning
+      Mais les évènements de type RebuildTriggered ne sont pas récupérés`, async () => {
+    // Arrange
+    const streamId = 'string|string';
+
+    const event1: DomainEvent = {
+      type: 'event-1',
+      payload: { test1: '1' },
+    };
+
+    const event2: DomainEvent = {
+      type: 'event-2',
+      payload: { test2: '2' },
+    };
+
+    const event3: RebuildTriggered = {
+      type: 'RebuildTriggered',
+      payload: { category: 'string', id: 'string' },
+    };
+
+    await publish('string|string', event1, event2, event3);
```

Comment:
pourquoi ne pas utiliser `streamId` ici ?


```suggestion
    await publish(streamId, event1, event2, event3);
```